### PR TITLE
Send unkown badge notification

### PR DESCRIPTION
**Pull request by mahdibenromdhane**

### CODE REVIEW

File: src/integration/pricing/export/convergent-charging/index.ts [hunk 1]
```diff
@@ -1 +1 @@
-export { default } from '../../dummy/DummyPricingIntegration';
+export { default } from '../../convergent-charging';
```
Comment (LucasBrazi06) [inline src/integration/pricing/export/convergent-charging/index.ts]: This file should not be changed

File: src/integration/refund/export/concur/index.ts [hunk 2]
```diff
@@ -1 +1 @@
-export { default } from '../../dummy/DummyRefundIntegration';
+export { default } from '../../concur';
```
Comment (LucasBrazi06) [inline src/integration/refund/export/concur/index.ts]: This file should not be changed

File: src/integration/smart-charging/export/sap-smart-charging/index.ts [hunk 3]
```diff
@@ -1 +1 @@
-export { default } from '../../dummy/DummySmartChargingIntegration';
+export { default } from '../../sap-smart-charging';
```
Comment (LucasBrazi06) [inline src/integration/smart-charging/export/sap-smart-charging/index.ts]: This file should not be changed

**Review by LucasBrazi06 — COMMENTED** [review]
In the end, no changes in this PR right?

**Comment by mahdibenromdhane** [comment]
> In the end, no changes in this PR right?

There is changes in 7 files

**Comment by LucasBrazi06** [comment]
> > In the end, no changes in this PR right?
> 
> There is changes in 7 files

Indeed!
They didn't show up in the first place :-( !
Thanks.

**Comment by LucasBrazi06** [comment]
@mahdibenromdhane Rename also the tag ID here.

**Pull request merged**